scripts/update/Updater.php

```diff
@@ -67,5 +67,13 @@ public function update($initialVersion)
         }
         
         $this->skip('0.1.0', '0.1.1');
+
+        if ($this->isVersion('0.1.1')) {
+            AclProxy::revokeRule(new AccessRule('grant', 'http://www.tao.lu/Ontologies/TAOTest.rdf#TestsManagerRole', array('ext'=>'taoQtiTestPreviewer', 'mod' => 'Previewer')));
```

Comment:
Where was this role set?
Can we use constants instead, might be easier to read

Reply:
Where should we store these kind of constants? As far as I see, there is no such constant defined yet.